libraries/csharp-net22-runtime/context/pipe

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+dotnet algo.dll
```

Comment:
This should work if the config.json works as I think it does...

Reply:
We should time `dotnet run`. If it's like `sbt run` then it takes a while because it's loading the whole dot net cli, instead of directly launching the algo.dll ? Not sure if theres a more direct way to load it for running?

Reply:
https://docs.microsoft.com/en-us/dotnet/core/deploying/

Reply:
Looks like using `dotnet algo.dll` is the prefered way, and will save space on the framework. Resolving this thread.

Reply:
yea I tried with dotnet run for a while, there is a ton of overhead and it requires literally pushing the entire project into the docker image, rather than just the dll and the configuration files